doc/bitcoin-conf.md

```diff
@@ -49,4 +49,12 @@ rpcport=4000
 
 The configuration file is not automatically created; you can create it using your favorite text editor. By default, the configuration file name is `bitcoin.conf` and it is located in the Bitcoin data directory, but both the Bitcoin data directory and the configuration file path may be changed using the `-datadir` and `-conf` command-line options.
 
+### Default configuration file locations
+
+Operating System | Data Directory | Path
```

Comment:
Why have `Data Directory`? `Path` is enough with env var.

Reply:
I agree. We could just have `macOS | $HOME/Library/Application Support/Bitcoin/`.
Especially since `$HOME` and `/Users/username` etc are the same thing. Seems a bit confusing to be using both.

Reply:
some people don't know what environment variables are. I think it's best if we leave both examples